ARM/ARM64 version panics

```
./workloader-linux-arm wkld-export
panic: interface conversion: interface {} is nil, not bool

goroutine 1 [running]:
github.com/brian1917/workloader/utils.GetPCEbyNameV2({0x100f010, 0xb}, 0x0)
	/Users/alex.goller/git/workloader/utils/pcev2.go:64 +0x908
github.com/brian1917/workloader/utils.GetTargetPCEV2(0x0)
	/Users/alex.goller/git/workloader/utils/pcev2.go:26 +0x1f0
github.com/brian1917/workloader/cmd/wkldexport.init.func1(0xea22b0, {0xeb7e70, 0x0, 0x0})
	/Users/alex.goller/git/workloader/cmd/wkldexport/cmd.go:60 +0xa4
github.com/spf13/cobra.(*Command).execute(0xea22b0, {0xeb7e70, 0x0, 0x0})
	/Users/alex.goller/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:944 +0x744
github.com/spf13/cobra.(*Command).ExecuteC(0xe9afb0)
	/Users/alex.goller/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x410
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/alex.goller/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
github.com/brian1917/workloader/cmd.Execute()
	/Users/alex.goller/git/workloader/cmd/root.go:292 +0x20
main.main()
	/Users/alex.goller/git/workloader/main.go:131 +0xfc0
```

pce.yaml file
```
continue_on_error: false
debug: false
default-pce:
    disable_tls: false
    fqdn: redacted
    key: redacted
    org: redacted
    port: 443
    user: redacted
default_pce_name: default-pce
log_file: workloader.log
max_entries_for_stdout: 100
no_prompt: false
output_format: csv
target_pce: ""
update_pce: false
verbose: false
```

--- a/utils/pce.go
+++ b/utils/pce.go
@@ -14,10 +14,10 @@ func GetTargetPCE(GetLabelMaps bool) (illumioapi.PCE, error) {
 
 	// Get the PCE name
 	var name string
-	if viper.Get("target_pce") != nil && viper.Get("target_pce").(string) != "" {
-		name = viper.Get("target_pce").(string)
-	} else if viper.Get("default_pce_name") != nil && viper.Get("default_pce_name").(string) != "" {
-		name = viper.Get("default_pce_name").(string)
+	if viper.GetString("target_pce") != "" {
+		name = viper.GetString("target_pce")
+	} else if viper.GetString("default_pce_name") != "" {
+		name = viper.GetString("default_pce_name")
 	} else {
 		LogError("there is no pce set using the --pce flag and there is no default pce. either run workloader pce-add to add your first pce or workloader set-default to set an existing PCE as default.")
 	}
@@ -61,9 +61,17 @@ func GetTargetPCE(GetLabelMaps bool) (illumioapi.PCE, error) {
 func GetPCEbyName(name string, GetLabelMaps bool) (illumioapi.PCE, error) {
 	var pce illumioapi.PCE
 	if viper.IsSet(name + ".fqdn") {
-		pce = illumioapi.PCE{FriendlyName: name, FQDN: viper.Get(name + ".fqdn").(string), Port: viper.Get(name + ".port").(int), Org: viper.Get(name + ".org").(int), User: viper.Get(name + ".user").(string), Key: viper.Get(name + ".key").(string), DisableTLSChecking: viper.Get(name + ".disableTLSChecking").(bool)}
-		if viper.Get(name+".proxy") != nil {
-			pce.Proxy = viper.Get(name + ".proxy").(string)
+		pce = illumioapi.PCE{
+			FriendlyName:       name,
+			FQDN:               viper.GetString(name + ".fqdn"),
+			Port:               viper.GetInt(name + ".port"),
+			Org:                viper.GetInt(name + ".org"),
+			User:               viper.GetString(name + ".user"),
+			Key:                viper.GetString(name + ".key"),
+			DisableTLSChecking: viper.GetBool(name + ".disableTLSChecking"),
+		}
+		if viper.GetString(name+".proxy") != "" {
+			pce.Proxy = viper.GetString(name + ".proxy")
 		}
 		if GetLabelMaps {
 			apiResps, err := pce.Load(illumioapi.LoadInput{Labels: true})
@@ -82,9 +90,17 @@ func GetPCEbyName(name string, GetLabelMaps bool) (illumioapi.PCE, error) {
 func GetPCENoAPI(name string) (illumioapi.PCE, error) {
 	var pce illumioapi.PCE
 	if viper.IsSet(name + ".fqdn") {
-		pce = illumioapi.PCE{FriendlyName: name, FQDN: viper.Get(name + ".fqdn").(string), Port: viper.Get(name + ".port").(int), Org: viper.Get(name + ".org").(int), User: viper.Get(name + ".user").(string), Key: viper.Get(name + ".key").(string), DisableTLSChecking: viper.Get(name + ".disableTLSChecking").(bool)}
-		if viper.Get(name+".proxy") != nil {
-			pce.Proxy = viper.Get(name + ".proxy").(string)
+		pce = illumioapi.PCE{
+			FriendlyName:       name,
+			FQDN:               viper.GetString(name + ".fqdn"),
+			Port:               viper.GetInt(name + ".port"),
+			Org:                viper.GetInt(name + ".org"),
+			User:               viper.GetString(name + ".user"),
+			Key:                viper.GetString(name + ".key"),
+			DisableTLSChecking: viper.GetBool(name + ".disableTLSChecking"),
+		}
+		if viper.GetString(name+".proxy") != "" {
+			pce.Proxy = viper.GetString(name + ".proxy")
 		}
 		return pce, nil
 	}
@@ -93,7 +109,7 @@ func GetPCENoAPI(name string) (illumioapi.PCE, error) {
 }
 
 func UseMulti() bool {
-	if viper.Get("get_api_behavior") == nil || viper.Get("get_api_behavior").(string) == "multi" {
+	if !viper.IsSet("get_api_behavior") || viper.GetString("get_api_behavior") == "multi" {
 		LogInfo("using multi get api behavior", false)
 		return true
 	}

--- a/utils/pcev2.go
+++ b/utils/pcev2.go
@@ -14,10 +14,10 @@ func GetTargetPCEV2(GetLabelMaps bool) (illumioapi.PCE, error) {
 
 	// Get the PCE name
 	var name string
-	if viper.Get("target_pce") != nil && viper.Get("target_pce").(string) != "" {
-		name = viper.Get("target_pce").(string)
-	} else if viper.Get("default_pce_name") != nil && viper.Get("default_pce_name").(string) != "" {
-		name = viper.Get("default_pce_name").(string)
+	if viper.GetString("target_pce") != "" {
+		name = viper.GetString("target_pce")
+	} else if viper.GetString("default_pce_name") != "" {
+		name = viper.GetString("default_pce_name")
 	} else {
 		LogError("there is no pce set using the --pce flag and there is no default pce. either run workloader pce-add to add your first pce or workloader set-default to set an existing PCE as default.")
 	}
@@ -61,9 +61,17 @@ func GetTargetPCEV2(GetLabelMaps bool) (illumioapi.PCE, error) {
 func GetPCEbyNameV2(name string, GetLabelMaps bool) (illumioapi.PCE, error) {
 	var pce illumioapi.PCE
 	if viper.IsSet(name + ".fqdn") {
-		pce = illumioapi.PCE{FriendlyName: name, FQDN: viper.Get(name + ".fqdn").(string), Port: viper.Get(name + ".port").(int), Org: viper.Get(name + ".org").(int), User: viper.Get(name + ".user").(string), Key: viper.Get(name + ".key").(string), DisableTLSChecking: viper.Get(name + ".disableTLSChecking").(bool)}
-		if viper.Get(name+".proxy") != nil {
-			pce.Proxy = viper.Get(name + ".proxy").(string)
+		pce = illumioapi.PCE{
+			FriendlyName:       name,
+			FQDN:               viper.GetString(name + ".fqdn"),
+			Port:               viper.GetInt(name + ".port"),
+			Org:                viper.GetInt(name + ".org"),
+			User:               viper.GetString(name + ".user"),
+			Key:                viper.GetString(name + ".key"),
+			DisableTLSChecking: viper.GetBool(name + ".disableTLSChecking"),
+		}
+		if viper.GetString(name+".proxy") != "" {
+			pce.Proxy = viper.GetString(name + ".proxy")
 		}
 		if GetLabelMaps {
 			apiResp, err := pce.GetLabels(nil)

